lib/puppet/provider/package/pkg.rb

```diff
@@ -23,7 +23,7 @@
 
   confine :osfamily => :solaris
 
-  defaultfor :osfamily => :solaris, :kernelrelease => '5.11'
+  defaultfor :osfamily => :solaris, :kernelrelease => ['5.11', '5.12']
```

Comment:
I think it would be good to get spec coverage for this change, ie it's the default for solaris 11 and 12, and not the default for solaris 10. It doesn't look like we have something checking this for 11 yet, but a similar example is here: https://github.com/puppetlabs/puppet/blob/master/spec/unit/provider/package/dnf_spec.rb#L8-L26